### PR TITLE
Add change in documentation to explain how to add collaborators in translation

### PR DESCRIPTION
**Pull request by claudiocebpaz**
Signed-off-by: claudio ceballos paz <claudio@hashdog.com>

#### Type of change

- Documentation update

#### Description

This shows in the docs how to add collaborators in the Hyperledger Org before they can be added to the translation team

### CODE REVIEW

File: docs/source/international_languages.md [hunk 1]
```diff
@@ -139,6 +139,9 @@ as spoken in Mexico:
    Once you've been added to this list, you can add others translators from your
    workgroup.
 
+1. On the [`i18n rocket channel`](https://chat.hyperledger.org/channel/i18n)
+   request adding collaborators if you want to able to assign issues. They need to become members of the community by asking core repo maintainers to add those translators to the [`Hyperledger Github Organization`](https://github.com/orgs/hyperledger/teamsr)
```
Comment (ale-linux) [inline docs/source/international_languages.md]: > if you want to able to assign

if you want to ^be able to assign

Comment (ale-linux) [inline docs/source/international_languages.md]: https://github.com/orgs/hyperledger/teamsr

it's a broken link

File: docs/source/international_languages.md [hunk 2]
```diff
@@ -139,6 +139,9 @@ as spoken in Mexico:
    Once you've been added to this list, you can add others translators from your
    workgroup.
 
+1. On the [`i18n rocket channel`](https://chat.hyperledger.org/channel/i18n)
```
Comment (ale-linux) [inline docs/source/international_languages.md]: Is a bulletpoint really needed here?

**Review by ale-linux — DISMISSED** [review]
Thanks for your contribution! A few minor nits

File: docs/source/international_languages.md (hunk 1 above)
Comment (claudiocebpaz) [reply docs/source/international_languages.md]: Done, Thanks!

Comment (claudiocebpaz) [reply docs/source/international_languages.md]: Done, Thanks!

File: docs/source/international_languages.md (hunk 2 above)
Comment (claudiocebpaz) [reply docs/source/international_languages.md]: I took the bullet point out. I was using the same structure, but this way is ok too.

**Comment by pamandrejko** [comment]
Thanks for adding this Claudio! What is the process you used to ask core repo maintainers to add those translators to the Hyperledger Github Organization? Is it through one of the RocketChat channels?

**Review by pamandrejko — APPROVED** [review]
Thanks for submitting this Claudio.

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.2

**Pull request merged**